frontend/src/app/utils/testResults.ts

```diff
@@ -43,15 +45,7 @@ export function hasPositiveFluResults(results: MultiplexResults): boolean {
     results.filter(
       (multiplexResult: MultiplexResult) =>
         multiplexResult.disease.name.includes("Flu") &&
-        getTestResult(multiplexResult) === "POSITIVE"
-    ).length > 0
-  );
-}
-
-export function hasCovidResults(results: MultiplexResults): boolean {
```

Comment:
⚠️ cleaned this up to use the `hasDiseaseSpecificResults` function in this file instead